bamboo/set-bamboo-env-variables.sh

```diff
@@ -199,5 +199,5 @@ if [[ $USE_CACHED_BOOTSTRAP == true ]]; then
 else
   export UNIT_TEST_BUILD_DIR=/uncached/cumulus
 fi
-
+export CUMULUS_UNIT_TEST_DATA=/tmp/cumulus_unit_test_data
```

Comment:
Nit: is this something that might ever need configuration?

Reply:
I don't see how or why, its only purpose is to be a temporary holder for these outputs
our docker-compose spec was already mounting it, although nothing was explicitly being dumped there that I could find

Reply:
although it just occurred to me that I should explicitly delete it the folder I'm putting in it, adding that